### PR TITLE
[IMPROVEMENT] Improve celeborn script logic

### PR DESCRIPTION
**Pull request by zwangsheng**
# [IMPROVEMENT] Improve celeborn script logic

### What changes were proposed in this pull request?
Improve celeborn scripts `celeborn-daemon.sh` and `celeborn-class.sh` print log and logic.

### What are the items that need reviewer attention?
`CELEBORN_PRINT_LAUNCH_COMMAND` was not used in `celeborn-daemon.sh`, move it to `celeborn-class.sh` and set default to 0 for help developer debug java command. 

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: sbin/celeborn-daemon.sh [hunk 1]
```diff
@@ -74,36 +74,34 @@ celeborn_rotate_log ()
...
-	while [ $num -gt 1 ]; do
-	    prev=`expr $num - 1`
-	    [ -f "$log.$prev" ] && mv "$log.$prev" "$log.$num"
-	    num=$prev
-	done
-	mv "$log" "$log.$num";
+	    while [ $num -gt 1 ]; do
+	      prev=`expr $num - 1`
+	      [ -f "$log.$prev" ] && mv "$log.$prev" "$log.$num"
+	      num=$prev
```
Comment (zwangsheng) [inline sbin/celeborn-daemon.sh:83]: Code behave differently in local and Github.

WX20221129-144133

Comment (waitinfuture) [reply sbin/celeborn-daemon.sh:83]: Seems there are tabs in the beginning of the line, we can replace with whitespace

Comment (waitinfuture) [reply sbin/celeborn-daemon.sh:83]: image

Comment (zwangsheng) [reply sbin/celeborn-daemon.sh]: done

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**